### PR TITLE
[google_maps_flutter] Add steps to set up google map sdk on each platform in readme.

### PR DESCRIPTION
**Pull request by cyanglaz**
People seem to have trouble stepping up Google Map SDKs(issue here: https://github.com/flutter/flutter/issues/30649). We should make our set up guide clearer.

### CODE REVIEW

File: packages/google_maps_flutter/README.md [hunk 1]
```diff
@@ -24,7 +24,18 @@ To use this plugin, add `google_maps_flutter` as a [dependency in your pubspec.y
...
+* Get an API key at <https://cloud.google.com/maps-platform/>.
+
+* Enable Google Map SDK for each platform.
+  * Go to [Google Developers Console](https://console.cloud.google.com/).
+  * Choose the project that you want to enable Google Maps on.
+  * Select the navigation menu and then select "Google Maps".
+  * Select "APIs" under the Google Maps menu.
+  * To enable Google Map for Android, select "Maps SDK for Android" in the "Additional APIs" section, then select "ENABLE".
+  * To enable Google Map for iOS, select "Maps SDK for iOS" in the "Additional APIs" section, then select "ENABLE".
+  * Make sure the APIs you enabled are under the "Enabled APIs" section.
```
Comment (cyanglaz) [inline packages/google_maps_flutter/README.md]: This has a potential to be a maintenance burden if the GDC undergoes a big update, which I doubt would happen every frequently. Should we mention that this steps might be out of date? cc @amirh

File: packages/google_maps_flutter/README.md [hunk 2]
```diff
@@ -24,7 +24,18 @@ To use this plugin, add `google_maps_flutter` as a [dependency in your pubspec.y
...
 
-Get an API key at <https://cloud.google.com/maps-platform/>.
+* Get an API key at <https://cloud.google.com/maps-platform/>.
+
+* Enable Google Map SDK for each platform.
+  * Go to [Google Developers Console](https://console.cloud.google.com/).
+  * Choose the project that you want to enable Google Maps on.
+  * Select the navigation menu and then select "Google Maps".
+  * Select "APIs" under the Google Maps menu.
+  * To enable Google Map for Android, select "Maps SDK for Android" in the "Additional APIs" section, then select "ENABLE".
```
Comment (amirh) [inline packages/google_maps_flutter/README.md]: nit: Google Maps

File: packages/google_maps_flutter/README.md [hunk 3]
```diff
@@ -24,7 +24,18 @@ To use this plugin, add `google_maps_flutter` as a [dependency in your pubspec.y
...
+  * To enable Google Map for iOS, select "Maps SDK for iOS" in the "Additional APIs" section, then select "ENABLE".
```
Comment (amirh) [inline packages/google_maps_flutter/README.md]: ditto

File: packages/google_maps_flutter/README.md [hunk 4]
```diff
@@ -24,7 +24,18 @@ To use this plugin, add `google_maps_flutter` as a [dependency in your pubspec.y
...
+  * Make sure the APIs you enabled are under the "Enabled APIs" section.
+
+* You can also find a detailed steps to get start with Google Maps Platform [here](https://developers.google.com/maps/gmp-get-started).
```
Comment (amirh) [inline packages/google_maps_flutter/README.md]: nit: "...find detailed steps"

**Pull request merged**